core/src/main/java/jenkins/util/ResourceBundleUtil.java

```diff
@@ -80,7 +80,7 @@ private ResourceBundleUtil() {
         ResourceBundle bundle = getBundle(baseName, locale, Jenkins.class.getClassLoader());
         if (bundle == null) {
             // Not in Jenkins core. Check the plugins.
-            Jenkins jenkins = Jenkins.getInstance(); // will never return null
+            Jenkins jenkins = Jenkins.get(); // will never return null
```

Comment:
We probably want to delete the if statement below because it can never be null, which will probably fix a spotbugs violation

Reply:
```suggestion
            Jenkins jenkins = Jenkins.getInstanceOrNull();
```